requirements-dev.txt

```diff
@@ -86,6 +86,9 @@ coverage==4.5.2 \
     --hash=sha256:ed02c7539705696ecb7dc9d476d861f3904a8d2b7e894bd418994920935d36bb \
```

Comment:
Unless the bumps are actually required by a change you made in this commit, could you make this a separate commit?

Reply:
I fixed that one.